src/Agent.Worker/ExecutionContext.cs

```diff
@@ -57,6 +59,8 @@ public sealed class ExecutionContext : AgentService, IExecutionContext
         private Guid _mainTimelineId;
         private Guid _detailTimelineId;
         private int _childExecutionContextCount = 0;
+        private TimeSpan timeout;
+        private CancellationTokenSource timeoutTokenSource;
```

Comment:
Can you fix these names to be prefixed with an `_`? I.e. `_timeout` and `_timeoutTokenSource`